data/pigui/modules/saveloadgame.lua

```diff
@@ -45,8 +39,7 @@ local function getSaveTooltip(name)
 	local ret
 	local stats
 	if not saveFileCache[name] then
-        local ok
-        ok, saveFileCache[name] = pcall(Game.SaveGameStats, name)
+        _, saveFileCache[name] = pcall(Game.SaveGameStats, name)
```

Comment:
This does actually wind up assigning to the global `_` variable, although that variable is not used by anything. It's weird, but I suppose there's not that much harm in it other than an extra roundtrip through the environment table.